test/unittests/api/v0.0.1/Projects_Test.php

```diff
@@ -1,226 +0,0 @@
-<?php
```

Comment:
Then what does it have to do with what I'm doing?
On 24 Sep 2015 10:02, "Dave MacFarlane" <notifications@github.com> wrote:

> In test/unittests/api/v0.0.1/Projects_Test.php
> <https://github.com/aces/Loris/pull/1257#discussion_r40321494>:
>
> > @@ -1,226 +0,0 @@
> > -<?php
>
> It tests that the "GET /projects/" request from the REST API works as
> documented. It can't be removed. The getProjectsWithProject tests that it
> works correctly when the "useProjects" config setting is true.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/aces/Loris/pull/1257/files#r40321494>.
>